onnxmltools/convert/coreml/datatype.py

```diff
@@ -96,14 +96,18 @@ def make_value_info(value, batch_size=1):
     except Exception:
         raise Exception("Not a valid CoreML type")
 
+    doc_string = ''
```

Comment:
I think we can directly use value.shortDescription to initialize doc_string because we can get default value as least.